Makefile

```diff
@@ -36,7 +36,7 @@ run:
 	./$(BINARY_NAME)
 
 deps:
-	dep ensure
+	@go get ./...
```

Comment:
We should change this to `GO111MODULE=on go mod download` so that we explicitly use go modules.
Otherwise we get this warning with GOPATH enabled,
```-> % go get ./...
go get: warning: modules disabled by GO111MODULE=auto in GOPATH/src;
	ignoring go.mod;
	see 'go help modules'
```

Reply:
Ahh yes that's a good point! Normally I do not work on things inside any `$GOPATH` and so avoid this situation altogether. But if we're not keen on changing where we clone the repo to in CI then yeah I agree.

Reply:
Done